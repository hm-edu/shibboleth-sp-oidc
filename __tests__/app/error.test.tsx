import { afterEach, beforeEach, describe, expect, it } from 'vitest';
import { cleanup, render, screen } from '@testing-library/react';
import Error from '@/app/error';

describe('Error', () => {
  beforeEach(() => {
    render(Error());
  });

  it('should exists', () => {
    expect(screen.findByTestId('error-illustration')).toBeDefined();
  });

  it('should match snapshot', () => {
    expect(screen.findByTestId('error-illustration')).toMatchSnapshot();
  });

  afterEach(() => {
    cleanup();
  });
});
