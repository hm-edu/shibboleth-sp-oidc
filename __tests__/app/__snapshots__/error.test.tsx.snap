// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Error > should match snapshot 1`] = `Promise {}`;
